components/DataRangePicker/index.js

```diff
@@ -101,10 +102,12 @@ class DataRangePicker extends React.Component {
             })}
           />
         </div>
+        {(
```

Comment:
some debugging leftover?

Reply:
Ah, thanks for catching that! forgot to remove those parenthesis after removing the conditional.